### ISSUE TITLE
C-c C-t C-t on deftest form can cause stringp, nil type error

### ISSUE THREAD

**Issue by RickMoynihan** [clojure-emacs/cider#1608]
When running a test with `C-C C-t C-t` with the cursor inside a deftest form an error occurs.

## Expected behavior

Tests should run and return success or failure message/report.

## Actual behavior
`error in process filter: cider-propertize: Wrong type argument: stringp, nil`
`error in process filter: Wrong type argument: stringp, nil`

With `toggle-debug-on-error`

````
Debugger entered--Lisp error: (wrong-type-argument stringp nil)
  propertize(nil face font-lock-type-face)
  cider-propertize(nil ns)
  cider-test-echo-summary((dict "error" 0 "fail" 0 "ns" 1 "pass" 0 "test" 0 "var" 1) (dict))
  #[257 "\306\307\"\306\310\"\306\311\"\306\312\"\306\313\"\314\235\203$
````

## Steps to reproduce the problem

Some deftests work others don't.  I'm not sure exactly when this occurs, but I think it might happen when a test includes a failing test who's `is` call is in another namespace.

- Put cursor on a deftest form where the error occurs
- Run `C-C C-t C-t`.

### CIDER version information

```
CIDER 0.11.0 (Clojure 1.8.0, nREPL 0.2.12)
```

### Emacs version

E.g. 24.4.1

### Operating system

OSX 10.9.5

**Comment by jeffvalk** [comment]
if you have a minimal repro case, please post that code.

**Comment by bbatsov** [comment]
Seems the `ns` is nil for some reason.

**Comment by jeffvalk** [comment]
This is a duplicate of #1599, which was fixed by 3b95fef.
